<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardCenter</class>
 <widget class="QMainWindow" name="BoardCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CENTER</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/resources/panel_2.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>768</height>
    </size>
   </property>
   <widget class="SingleButton" name="start" native="true">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>114</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="retroceso" native="true">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>114</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="atoUp" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>114</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="atoDown" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>263</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="reconocer" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="puerta" native="true">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>310</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="desacople" native="true">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>514</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
   </widget>
   <widget class="SingleButton" name="senaleraAcople" native="true">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>514</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SingleButton</class>
   <extends>QWidget</extends>
   <header>singlebutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
